pkg/fqdn/name_manager_gc.go

```diff
@@ -50,7 +48,7 @@ func (n *NameManager) GC(ctx context.Context) error {
 		// marked active by the CT GC. Since we expire in this controller, we
 		// give these entries 2 cycles of TTL to allow for timing mismatches
 		// with the CT GC.
-		activeConnectionsTTL = int(2 * DNSGCJobInterval.Seconds())
+		activeConnectionsTTL = n.config.ActiveConnectionsTTL * int(DNSGCJobInterval.Seconds())
```

Comment:
What we are actually doing here is giving the user the ability to customize the multiplier used to set the TTL.

Following the referenced issue, I think we should instead setting `int(2 * DNSGCJobInterval.Seconds())` as a default value and then overwrite it with `n.config.ActiveConnectionsTTL` if the user given value is equal or greater (maybe logging something in case it isn't, to signal that the GC will use the default value).

Reply:
Yes, for sure if the configuration is a TTL then it should be typed as a time and not a multiplier. I doubt that we really want to expose this configuration knob actually as a multiplier, that just makes things even more confusing for the user.